packages/rule-manifest-is-valid/tests/tests.ts

```diff
@@ -56,42 +54,39 @@ const defaultTests: Array<RuleTest> = [
         }
     },
     {
-        name: `Web app manifest is specified and the 'background_color' and 'theme_color'' properties are not valid`,
-        reports: [
-            { message: `'background_color' property value ('invalid') is invalid` },
-            { message: `'theme_color' property value ('invalid') is invalid` }
-        ],
+        name: `Web app manifest is specified and the 'background_color' and 'theme_color'' properties are are valid`,
```

Comment:
Remove extra `'` at the end of `'theme_color'` and extra `are`

Reply:
Done.